docs/dom-testing-library/api-events.mdx

```diff
@@ -91,9 +91,9 @@ fireEvent.drop(getByLabelText(/drop files here/i), {
 element in the DOM and the key you want to fire.
 
 ```javascript
-fireEvent.keyDown(domNode, { key: 'Enter', code: 'Enter' })
+fireEvent.keyDown(domNode, { key: 'Enter', code: 'Enter', charCode: 13 })
```

Comment:
Thanks for the feedback. I dont' think we want to document deprecated event properties though (https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/charCode). Is the charCode important here for any framework?